client/src/test/resources/dom-setup.js

```diff
@@ -1,6 +1,21 @@
 /* eslint-disable */
 let jsdomMod = require("jsdom");
 
+// Ignore CSS imports required by Scala.js facades (e.g. Handsontable styles)
```

Comment:
What are these changes for?

Reply:
getting error while running `sbt --error client/test`